### PR TITLE
Add target cluster option to social event test util

### PR DESCRIPTION
**Pull request by alexeykazakov**
Part of https://issues.redhat.com/browse/SANDBOX-724

### CODE REVIEW

File: pkg/test/socialevent/socialevent_test.go [hunk 1]
```diff
@@ -90,4 +90,14 @@ func TestNewSocialEvent(t *testing.T) {
...
 
+	t.Run("with custom target cluster", func(t *testing.T) {
+		// when
+		e := testsocialevent.NewSocialEvent(test.HostOperatorNs, socialevent.NewName(), testsocialevent.WithTargetCluster("member-1"))
+		// then
+		assert.Equal(t, "deactivate30", e.Spec.UserTier)  // default
+		assert.Equal(t, "base1ns", e.Spec.SpaceTier)      // default
+		assert.Equal(t, 10, e.Spec.MaxAttendees)          // default
+		assert.Equal(t, 0, e.Status.ActivationCount)      // default
+		assert.Equal(t, "member-1", e.Spec.TargetCluster) // default
```
Comment (rajivnathan) [inline pkg/test/socialevent/socialevent_test.go:101]:
```suggestion
		assert.Equal(t, "member-1", e.Spec.TargetCluster) // custom
```

**Review by rajivnathan — APPROVED** [review]
Maybe worth adding an assertion for this field to the "default" values test? I guess the default is just empty?

**Comment by alexeykazakov** [comment]
> Maybe worth adding an assertion for this field to the "default" values test? I guess the default is just empty?

Yep. Good idea. Added here: https://github.com/codeready-toolchain/toolchain-common/pull/424/commits/dc89633a1e27bf349dfa42e9bd0b1dae6d9a567a
